### PR TITLE
Updating extensions for Rails SR, GTM Air NZ

### PR DESCRIPTION
**Pull request by j-ibrahim**
1) extension update for Rails SR page
2) updating extension for GTM Air NZ 
3) Clean up unused code

### CODE REVIEW

File: src/library/simplified/extension/rail-cards.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+if(utag.isRailSearchResults()) {
+ var cards = "";
+ if(b.entity.railSearch.searchParameters.cardOptions.length != undefined)
```
Comment (trungnguyenexp) [inline src/library/simplified/extension/rail-cards.js:3]: How do I get entity.railSearch.searchParameters.cardOptions to show :)? 

Here is my deeplink:
https://www.expedia.co.uk/rails/search?SearchBy=DEPARTURE&age_pax=40%2C40&dest=GBSEF&destinationName=Seaford%20(Sussex)%2C%20Seaford%2C%20UK&journeyType=oneway&orig=GB%23AS&originName=Ashford%20International%20(All%20Stations)%2C%20UK&sDate=10%2F05%2F2017&sTime=2300

And here is utag_data: 
image

File: src/library/simplified/extension/departure-month.js [hunk 2]
```diff
@@ -21,25 +24,3 @@
...
-    }
-
-    function getDepartureMonth(utagKey) {
-        if (utagKey) {
-            return new Date(utag_data[utagKey]).getMonth() + 1
-        }
-        return '';
-    }
-})();
- **/
```
Comment (trungnguyenexp) [inline src/library/simplified/extension/departure-month.js:45]: We can go ahead and remove old code :).

File: src/library/simplified/extension/currency-code.js [hunk 3]
```diff
@@ -68,70 +72,3 @@ else if(utag.isItinPage() && typeof b["entity.tripDetails.totalPrice"] != "undef
...
-}
- else if(utag.isLXCO() && utag_data.entity.checkout.cartTotal.currency){
-    utag_data["currencyCode"] = utag_data.entity.checkout.cartTotal.currency;
-}
- else if(utag.isCarCO() && typeof utag_data["entity.tripDetails.totalPrice"] != "undefined"){
-    utag_data['currencyCode'] = utag_data["entity.tripDetails.totalPrice.currency"] ? utag_data["entity.tripDetails.totalPrice.currency"] : "";
-}
- else if(utag.is3pp()){
-  utag_data['currencyCode'] = utag_data["entity.checkout.currency"];
-}
```
Comment (trungnguyenexp) [inline src/library/simplified/extension/currency-code.js:136]: We can go ahead and remove old code :).

**Review by trungnguyenexp — APPROVED** [review]

File: src/library/simplified/extension/currency-code.js (hunk 3 above)
Comment (j-ibrahim) [reply src/library/simplified/extension/currency-code.js:136]: Commented old codes are removed.

File: src/library/simplified/extension/departure-month.js (hunk 2 above)
Comment (j-ibrahim) [reply src/library/simplified/extension/departure-month.js:45]: Commented old codes are removed.

File: src/library/simplified/extension/rail-cards.js (hunk 1 above)
Comment (j-ibrahim) [reply src/library/simplified/extension/rail-cards.js:3]: https://www.expedia.co.uk/rails/search?SearchBy=DEPARTURE&age_pax=40%2C40&dest=GBSEF&destinationName=Seaford%20(Sussex)%2C%20Seaford%2C%20UK&journeyType=oneway&orig=GB%23AS&originName=Ashford%20International%20(All%20Stations)%2C%20UK&rail_card=UK_ANNUAL_GOLD&sDate=10%2F05%2F2017&sTime=2300&search=playback

image

**Pull request merged**